dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -199,32 +199,34 @@ def load_section
     end
   end
 
-  def select_level
+  def select_levels
     # If there's only one level in this scriptlevel, use that
-    return @script_level.levels[0] if @script_level.levels.length == 1
+    return @script_level.levels if @script_level.levels.length == 1
 
-    # For teachers, load the student's most recent attempt
```

Comment:
The level selection logic for teachers is now the same as it is for students, but I forgot about the difference between `@user` and current_user. Added a line back to handle that.